code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -660,6 +660,18 @@
 
 /obj/item/borg/upgrade/transform/clown
 	name = "borg module picker (Clown)"
-	desc = "Allows you to to turn a cyborg into a clown, honk."
+	desc = "Allows you to turn a cyborg into a clown, honk."
 	icon_state = "cyborg_upgrade3"
 	new_module = /obj/item/robot_module/clown
+
+/obj/item/borg/upgrade/transform/security
+	name = "borg module picker (Security)"
+	desc = "Allows you to turn a cyborg into a validhunter, HALT!"
```

Comment:
'validhunter' is ick ock

Reply:
no its ock ick

Reply:
Just change it to "crime fighter"

Reply:
Fixed

Reply:
I liked "validhunter" better as a humorous comment but alright.
"hunter" is weird